HousingSearchApi/V2/Gateways/SearchGateway.cs

```diff
@@ -44,8 +55,19 @@ public async Task<SearchResponseDto> Search(string indexName, SearchParametersDt
         };
     }
 
+
+    private Func<QueryContainerDescriptor<object>, QueryContainer>
+        MatchPhrasePrefix(string searchText, double boost, string fieldName) =>
+        should => should
+            .MatchPhrasePrefix(mp => mp
```

Comment:
May still be an issue if their search term is not the start of the string.
But better than nothing at this very moment.